compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs

```diff
@@ -88,6 +88,7 @@ data PackageConfigFields = PackageConfigFields
     , pExposedModules :: Maybe [String]
     , pVersion :: Maybe String
     , pDependencies :: [String]
+    , pDataImports :: [String]
```

Comment:
Given that the other field is called `dependencies` maybe `data-dependencies` would make sense? @bame-da what do you think?